integration/obscuroscan/obscuroscan_test.go

```diff
@@ -93,19 +103,28 @@ func TestObscuroscan(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, 200, statusCode)
 
-	statusCode, body, err = fasthttp.Get(nil, fmt.Sprintf("%s/items/transactions/", serverAddress))
+	statusCode, body, err = fasthttp.Get(nil, fmt.Sprintf("%s/items/transactions/?offset=0&size=99", serverAddress))
 	assert.NoError(t, err)
 	assert.Equal(t, 200, statusCode)
 
 	type publicTxsRes struct {
-		Result []common.PublicTxData `json:"result"`
+		Result common.PublicQueryResponse `json:"result"`
 	}
 
 	publicTxsObj := publicTxsRes{}
 	err = json.Unmarshal(body, &publicTxsObj)
 	assert.NoError(t, err)
-	assert.Equal(t, 1, len(publicTxsObj.Result))
+	assert.Equal(t, 1, len(publicTxsObj.Result.PublicTxData))
+	assert.Equal(t, uint64(1), publicTxsObj.Result.Total)
+
+	issueTransactions(
+		t,
+		fmt.Sprintf("ws://127.0.0.1:%d", startPort+integration.DefaultHostRPCWSOffset),
+		wallet.NewInMemoryWalletFromConfig("8dfb8083da6275ae3e4f41e3e8a8c19d028d32c9247e24530933782f2a05035b", integration.ObscuroChainID, testlog.Logger()),
+		100,
+	)
 
+	// time.Sleep(time.Hour)
```

Comment:
this comment can be removed (unless you use it often to test things locally)

Reply:
Yes, I typically test with that. And because the test isn't enabled it's probably fine to keep it there for now.